src/Lykke.RabbitMqBroker/RabbitMqSubscriptionSettings.cs

```
@@ -133,6 +225,25 @@ private static string GetDeadLetterExchangeName(string @namespace, string nameOf
             return $"{NormalizeName(@namespace)}.{NormalizeName(endpointName)}.{NormalizeName(nameOfSourceEndpoint)}.dlx";
         }
 
+        private static string GetExchangeNameWithValidation(string @namespace, string endpointName)
+        {
+            if (@namespace == LykkeNameSpace && endpointName.StartsWith($"{LykkeNameSpace}."))
+                return $"{NormalizeName(endpointName)}";
```

I prefer to throw exception in this case, since it's invalid usage of the library.